js_src/rdm.js

```diff
@@ -5,146 +5,834 @@ angular.module('rdmApp', [])
   $interpolateProvider.endSymbol('}]}');
  }])
 
- .controller('UIDController', ['$scope', function(convertor) {
+ .constant('RDM', {
+  'COMMAND_CLASS': {
+   'DISCOVERY_COMMAND': 16,
+   'DISCOVERY_COMMAND_RESPONSE': 17,
+   'GET_COMMAND': 32,
+   'GET_COMMAND_RESPONSE': 33,
+   'SET_COMMAND': 48,
+   'SET_COMMAND_RESPONSE': 49
+  },
+  'EUID_SIZE': 16,
+  'NACK_REASON': {
+   'NR_UNKNOWN_PID': 0,
+   'NR_FORMAT_ERROR': 1,
+   'NR_HARDWARE_FAULT': 2,
+   'NR_PROXY_REJECT': 3,
+   'NR_WRITE_PROTECT': 4,
+   'NR_UNSUPPORTED_COMMAND_CLASS': 5,
+   'NR_DATA_OUT_OF_RANGE': 6,
+   'NR_BUFFER_FULL': 7,
+   'NR_PACKET_SIZE_UNSUPPORTED': 8,
+   'NR_SUB_DEVICE_OUT_OF_RANGE': 9,
+   'NR_PROXY_BUFFER_FULL': 10,
+   'NR_ACTION_NOT_SUPPORTED': 11
+  },
+  'PIDS': {
```

Comment:
That's the next version - loading the PID information using AJAX. We need to build an index on the server side